.travis.yml

```diff
@@ -14,10 +14,5 @@ matrix:
       dist: xenial
     - python: nightly
       dist: xenial
-# TODO: https://bugs.python.org/issue40334
-# the PEG parser is currently broken in some cases
-  allow_failures:
-    - python: nightly
-      dist: xenial
```

Comment:
oh oops now that I look at this, this is 3.10a0 -- should we select `3.9` here explicitly?

Reply:
I've set it to `3.9-dev`. It seems to be the best way at the moment.